skribenten-web/frontend/src/Brevredigering/LetterEditor/components/ContentGroup.tsx

```diff
@@ -244,18 +262,50 @@ export function EditableText({ literalIndex, content }: { literalIndex: LiteralI
     }
   };
 
+  const handleOnclick = (e: React.MouseEvent) => {
```

Comment:
Har ikke testet, men det er ikke problematisk å kalle på `preventDefault` for elementer som ikke er fritekst?

Reply:
Alt ser ut til å fungere som det skal

Reply:
Jeg var bare redd for at vi fjernet mulighet for å klikke et sted for å plassere tekstmarkøren.